en_ewt-ud-train.conllu

```diff
@@ -158529,7 +158529,7 @@
 21	the	the	DET	DT	Definite=Def|PronType=Art	23	det	23:det	_
 22	early	early	ADJ	JJ	Degree=Pos	23	amod	23:amod	_
 23	60s	60	NOUN	NNS	Number=Plur	17	obl	17:obl:in	_
-24	use	use	VERB	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	ccomp	6:ccomp	ToCheck=Number
+24	use	use	VERB	VBP	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	ccomp	6:ccomp	_
```

Comment:
VerbForm=Inf? I guess the subjunctive is a special use of the infinitive form.

Reply:
OK